docs/en/data_types/datetime.md

```diff
@@ -1,45 +1,41 @@
 # DateTime {#data_type-datetime}
 
-Data structure storing Unix timestamp. Also, it can store a time zone.
+Data structure for storing [Unix time](https://en.wikipedia.org/wiki/Unix_time). It can also store a time zone.
```

Comment:
"timestamp" is a proper word here, "time" is some rules how these timestamps are supposed to be treated.
Meanwhile when there's a timezone it stops being Unix time. Needs to be fixed in all following text.